### PR TITLE
fix(sbb-train): improve accessibility by replacing `span` with heading tag

### LINKED ISSUE #1771: [Bug]: SbbTrainFormation - direction ignored by SR
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/master/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/orgs/lyne-design-system/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Accessibility

### It affects the following packages

Components

### Which version of Lyne Components are you using

0.33.10

### Operating system

Windows 10

### Browser / Browser version

Chrome

### Input mode

Mouse/Keyboard

### Additional settings

_No response_

### What happened?

Screenreaders ignores the direction of the train (Tested with nvda and voiceover)

### Steps to reproduce the issue

1. Navigate trought the train-formation with the arrow keys

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by jeripeierSBB**
Closes #1771

### CODE REVIEW

File: src/components/sbb-train/sbb-train.tsx [hunk 1]
```diff
@@ -30,6 +31,9 @@ export class SbbTrain implements ComponentInterface {
   /** General label for "driving direction". */
   @Prop() public directionLabel!: string;
 
+  /** Heading level of the direction label, used for screenreaders. */
+  @Prop() public directionLabelLevel: InterfaceTitleAttributes['level'] = '3';
```
Comment (jeripeierSBB) [inline src/components/sbb-train/sbb-train.tsx:35]:
```suggestion
  @Prop() public directionLabelLevel: InterfaceTitleAttributes['level'] = '6';
```

File: src/components/sbb-train/sbb-train.spec.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ describe('sbb-train', () => {
       <sbb-train direction-label="Driving direction" station="Bern" direction="left">
         <mock:shadow-root>
           <div class="sbb-train">
-            <span class="sbb-screenreaderonly">Train, Driving direction Bern.</span>
+            <sbb-title visually-hidden level="6">Train, Driving direction Bern.</sbb-title>
```
Comment (kyubisation) [inline src/components/sbb-train/sbb-train.spec.ts:15]: Is it necessary to use an `sbb-title` here or would a h1-6 be enough?

**Review by kyubisation — APPROVED** [review]
LGTM 👍 
One question about title usage

File: src/components/sbb-train/sbb-train.spec.ts (hunk 2 above)
Comment (jeripeierSBB) [reply src/components/sbb-train/sbb-train.spec.ts:15]: @kyubisation No it's not necessary, my thought was the chance that the title component is anyways loaded in a page context is very high and so there is less payload than all the rules of the screenreader only mixin. But I can easily change it if you want?

**Pull request merged**